cooperative_lanechange/src/cooperative_lanechange.cpp

```diff
@@ -589,17 +613,20 @@ namespace cooperative_lanechange
         std::vector<double> times;
         trajectory_utils::conversions::speed_to_time(downtracks, final_actual_speeds, &times);
 
+        // Remove extra points
+        ROS_DEBUG_STREAM("Before removing extra buffer points, future_geom_points.size()" << future_geom_points.size());
+        int end_dist_pt_index = basic_autonomy::waypoint_generation::get_nearest_index_by_downtrack(future_geom_points, wm_, ending_state_before_buffer_);
+        future_geom_points.resize(end_dist_pt_index);
+        times.resize(end_dist_pt_index);
+        final_yaw_values.resize(end_dist_pt_index);
+        ROS_DEBUG_STREAM("After removing extra buffer points, future_geom_points.size()" << future_geom_points.size());
+
         // Build trajectory points
         // TODO When more plugins are implemented that might share trajectory planning the start time will need to be based
```

Comment:
Looks like this was fixed by PR#1088 https://github.com/usdot-fhwa-stol/carma-platform/pull/1088.
This comment can be removed.

Reply:
Deleted